vue-touch.js

```diff
@@ -66,7 +66,9 @@
         // define new handler
         this.handler = function (e) {
           e.targetVM = vm
+          vm.$event = e
```

Comment:
Wouldn't it be interesting to store the previous value of vm.$event (if any), and restore it afterwards ? This could avoid some issues with other pieces of code that might use $event.